tests/src/test/scala/cats/tests/VarianceSuite.scala

```diff
@@ -23,24 +25,45 @@ class VarianceSuite extends CatsSuite {
   }
 
   test("Auto-variance should widen a bifunctor automatically") {
-    def shouldInfer[F[_, _]: Bifunctor](fi: F[Int, Int]) =
+    def shouldInfer[F[_, _]: Bifunctor](fi: F[Int, Int]): F[Either[Bar, Baz.type], Either[Bar, Baz.type]] =
       fi.bimap(i => if (true) Left(Bar(i)) else Right(Baz), i => if (true) Left(Bar(i)) else Right(Baz))
 
     def inferred[F[_, _]: Bifunctor](fi: F[Int, Int]): F[Either[Foo, Foo], Either[Foo, Foo]] = shouldInfer[F](fi)
   }
 
   test("Auto-variance should left widen a bifunctor automatically") {
-    def shouldInfer[F[_, _]: Bifunctor](fi: F[Int, Int]) =
+    def shouldInfer[F[_, _]: Bifunctor](fi: F[Int, Int]): F[Int, Either[Bar, Baz.type]] =
       fi.bimap(identity, i => if (true) Left(Bar(i)) else Right(Baz))
 
     def inferred[F[_, _]: Bifunctor](fi: F[Int, Int]): F[Int, Either[Foo, Foo]] = shouldInfer[F](fi)
   }
 
   test("Auto-variance should right widen a bifunctor automatically") {
-    def shouldInfer[F[_, _]: Bifunctor](fi: F[Int, Int]) =
+    def shouldInfer[F[_, _]: Bifunctor](fi: F[Int, Int]): F[Either[Bar, Baz.type], Int] =
       fi.bimap(i => if (true) Left(Bar(i)) else Right(Baz), identity)
 
     def inferred[F[_, _]: Bifunctor](fi: F[Int, Int]): F[Either[Foo, Foo], Int] = shouldInfer[F](fi)
   }
 
+  test("Auto-variance should widen a profunctor automatically") {
```

Comment:
right-widen, maybe?

Reply:
I agree. If we don't want to be specific about the side we could also stay general like
```suggestion
  test("Auto-variance should convert a profunctor's variance automatically") {
```